app/src/main/java/woowacourse/paint/PaintBinding.kt

```diff
@@ -0,0 +1,37 @@
+package woowacourse.paint
+
+import androidx.databinding.BindingAdapter
+import com.google.android.material.slider.RangeSlider
+
+@BindingAdapter("app:setColorPalette")
+fun BoardView.setColorPalette(colorPalette: ColorPalette?) {
+    if (colorPalette == null) return
+    changeColor(colorPalette)
+}
+
+@BindingAdapter("app:setWidth")
+fun BoardView.setWidth(width: Float?) {
+    if (width == null) return
+    setWidth(width)
+}
+
+@BindingAdapter("app:eraseAll")
+fun BoardView.eraseAll(isErasing: Boolean?) {
+    if (isErasing == null) return
+    if (isErasing) erase()
+}
+
+@BindingAdapter("app:setupSlider")
+fun RangeSlider.setupSlider(viewModel: MainViewModel) {
+    this.valueFrom = MIN_WIDTH_VALUE
+    this.valueTo = MAX_WIDTH_VALUE
```

Comment:
```suggestion
activity_main.xml
<com.google.android.material.slider.RangeSlider
    ...
    android:valueFrom="0.0"
    android:valueTo="100.0" />
```
요런식으로 xml에서 바로 설정할 수 있습니당